### PR TITLE
Pylint alerts corrections as part of an intervention experiment 40

### PR DESCRIPTION
**Pull request by evidencebp**
Makes the interventions describe in [intervention issue](https://github.com/mralext20/alex-bot/issues/40).
The experiment is described [here](https://github.com/evidencebp/pylint-intervention/).

Each intervention was done in a dedicated commit with a message explaining it.

### CODE REVIEW

File: alexBot/cogs/utils.py [hunk 1]
```diff
@@ -41,7 +41,10 @@ async def roll(self, interaction: discord.Interaction, dice: str):
                     return await interaction.response.send_message(
                         "You can't roll more than 100 dice at once!", ephemeral=True
                     )
-            except Exception:
+            except (TypeError
+                    , ValueError
+                    , discord.HTTPException
+                    , discord.InteractionResponded):
```
Comment (xela-zone) [inline alexBot/cogs/utils.py:47]: not necessary to be checked for, as the interaction will have not been responded too yet


```suggestion
):
```

**Review by xela-zone — APPROVED** [review]
these look great! 

gonna add some further changes myself too

**Comment by evidencebp** [comment]
I liked what you added.
Can you merge now?

**Comment by evidencebp** [comment]
Thanks, @mralext20 !

**Pull request merged**